__tests__/api/chat/channels/read.test.js

```diff
@@ -0,0 +1,91 @@
+const supertest = require('supertest');
+const app = require('../../../../app');
+const generateUserAndFollowSeeds = require('../../__utils__/seeds/users_and_follow_seeds');
+const {
+  createReusableAuthTestSuite,
+  createReusablePreventAnonymousUserTestSuite
+} = require('../../__authTest__/createReusableAuthTestSuite');
+
+beforeEach(() => {
+  generateUserAndFollowSeeds();
+});
+
+describe('POST /chat/channels/read', () => {
+  createReusableAuthTestSuite(
+    supertest(app)
+      .post('/api/v1/chat/channels/read')
+      .send({channelId: 'channel-id', channelType: 1})
+  );
+  createReusablePreventAnonymousUserTestSuite(
+    supertest(app)
+      .post('/api/v1/chat/channels/read')
+      .set({Authorization: 'Bearer anon-token'})
+      .send({channelId: 'channel-id', channelType: 1})
+  );
+
+  test('should return 403 error validation if channelId is not provided', async () => {
+    const response = await supertest(app)
+      .post('/api/v1/chat/channels/read')
+      .set({Authorization: `Bearer token`})
+      .send({
+        channelType: 1
+      })
+      .expect(403);
+
+    expect(response.body).toEqual(
+      expect.objectContaining({
+        status: 'error validation',
+        message: expect.arrayContaining([
+          expect.objectContaining({
+            field: 'channelId',
+            message: expect.any(String)
+          })
+        ])
+      })
+    );
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In this test case, you are checking if the server returns a 403 error when `channelId` is not provided. However, it would be better to check for a 400 (Bad Request) error instead of a 403 (Forbidden) error. A 400 error indicates that the server could not understand the request due to invalid syntax, which is more appropriate in this case.

```diff
-       .expect(403);
+       .expect(400);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->